tools/cleanup/cleanup.sh

```diff
@@ -69,42 +69,61 @@ delete_tvk_op() {
       fi
     fi
 
-    # Delete k8s-triliovault-resource-cleaner cronjob
-    tvkcron=$(kubectl get cronjob --no-headers -n openshift-operators 2>/dev/null | grep k8s-triliovault | awk '{print $1}')
-    if [ -n "${tvkcron}" ]; then
-      echo "Deleting k8s-triliovault-resource-cleaner cronjob"
-      kubectl delete cronjob "${tvkcron}" -n openshift-operators
-      retValue=$?
-      if [ "${retValue}" -ne 0 ]; then
-        exit_status=1
-      fi
-    fi
   fi
 
   # For Upstream OR in case if TVK installed on OCP using "helm"
   # Delete Triliovault-manager and Triliovault-operator using helm/label
-  if (helm list -A | grep -v REVISION | grep triliovault >/dev/null 2>&1); then
-    echo "Uninstalling Trilivault-manager"
-    tvm=$(helm list -A | grep -v REVISION | grep triliovault-v | awk '{print $1}')
-    tvm_ns=$(helm list -A | grep -v REVISION | grep triliovault-v | awk '{print $2}')
-    if [ -n "${tvm}" ]; then
-      helm uninstall "${tvm}" -n "${tvm_ns}"
+  # Fetch non-deuplicate namespace
+  tvm_ns=$(helm list -A | grep -v REVISION | grep 'triliovault' | awk '{print $2}' | uniq)
+  if [ -n "${tvm_ns}" ]; then
+    for ns in ${tvm_ns}; do
+
+      # Deleting Trilivault-manager CR
+      tvm_name=$(kubectl get triliovaultmanager --no-headers -n "${ns}" | awk '{print $1}')
+      echo "Patching triliovaultmanager CR ${tvm_name} in namespace ${ns}"
+      kubectl patch triliovaultmanager "${tvm_name}" -p '{"metadata":{"finalizers":[]}}' --type=merge -n "${ns}"
+      echo "Deleting triliovaultmanager CR ${tvm_name} in namespace ${ns}"
+      kubectl delete triliovaultmanager "${tvm_name}" -n "${ns}"
       retValue=$?
       if [ "${retValue}" -ne 0 ]; then
         exit_status=1
       fi
-    fi
-    echo "Uninstalling Trilivault-operator"
-    tvo=$(helm list -A | grep -v REVISION | grep triliovault-o | awk '{print $1}')
-    tvo_ns=$(helm list -A | grep -v REVISION | grep triliovault-o | awk '{print $2}')
-    if [ -n "${tvo}" ]; then
-      helm uninstall "${tvo}" -n "${tvo_ns}"
-      retValue=$?
-      if [ "${retValue}" -ne 0 ]; then
-        exit_status=1
+
+      # Uninstall Trilivault-manager
+      tvm=$(helm list -n "${ns}" | grep -v REVISION | grep 'triliovault-[0-9]' | awk '{print $1}')
```

Comment:
`tvm` can contain multiple values, need to handle this loop as well

Reply:
Not addressed as discussed - k8s-triliovault  SA name will have clash in same ns..